docs/cli/00-overview.md

```diff
@@ -23,6 +23,37 @@ For Mac OS users, you will need to allow `mass` to run by opening the `System Pr
 
 :::
 
+### Setup
+
+In order to execute commands against your Massdriver organization, you must set environment variables for the CLI to use.
+
+#### Find your organization ID
+
+To find your organization ID, hover over your organization name logo in the top left corner of the Massdriver UI and click the copy button next to your organization ID.
+
+#### Use your organization ID in the Massdriver CLI
+
+To use your organization ID in the Massdriver CLI, export the `MASSDRIVER_ORG_ID` environment variable to the value of your Organization UUID.
+
+```bash
+export MASSDRIVER_ORG_ID=your-org-id
```

Comment:
I think we have a screen cap someplace in this repo of where to find this

Reply:
In /concepts